users/api/querying-pinot-using-standard-sql/response-format.md

```diff
@@ -187,19 +187,137 @@ $ curl -X POST \
-
-
-
-
-
-
-
+<table>
```

Comment:
The content of the table is almost the same, but it has been formatted in order to be readable in the source code.

The only change is in the last row, where `traceInfo` has been substituted with `stageStats`